### PR TITLE
YLP-115 Add intermediate screen after login to renew consent

### PR DESCRIPTION
**Pull request by yvon-dblg**
image

### CODE REVIEW

File: locales/en/translation.json [hunk 1]
```diff
@@ -702,11 +699,13 @@
...
   "You must be at least 13 years old.": "You must be at least 13 years old.",
+  "You need to accept the changes to continue.": "You need to accept the changes to continue.",
   "You own your data. Read all the details about Tidepool's Big Data Donation project": "You own your data.",
   "You want to manage your diabetes data. You are caring for or supporting someone with diabetes.": "You want to manage your diabetes data. You are caring for or supporting someone with diabetes.",
   "You will no longer be able to see or comment on their data.": "You will no longer be able to see or comment on their data.",
   "You've been invited to Tidepool.": "You've been invited to YourLoops.",
   "Your account has been locked for {{delayBeforeNextLoginAttempt}} minutes. You have reached the maximum number of login attempts.": "Too many failed login attempts. Please try again in {{delayBeforeNextLoginAttempt}} minutes.",
+  "Your need to accept the Terms of Use and Privacy Policy of {{brand}} to continue.": "Your need to accept the Terms of Use and Privacy Policy of {{brand}} to continue.",
```
Comment (franck-fourel) [inline locales/en/translation.json:708]:
```suggestion
  "You need to accept the Terms of Use and Privacy Policy of {{brand}} to continue.": "You need to accept the Terms of Use and Privacy Policy of {{brand}} to continue.",
```

File: app/pages/terms/terms.js [hunk 2]
```diff
@@ -13,281 +13,173 @@
...
+          <button
+              className="btn btn-primary"
+              onClick={this.handleTermsAndPrivacySubmit.bind(this)}
+              disabled={!agreed}>
+            {t('Continue')}
+          </button>
+          <button
+              className="btn btn-secondary"
+              onClick={this.handleRefuseTerms.bind(this)}>
+            {t('Refuse')}
```
Comment (franck-fourel) [inline app/pages/terms/terms.js:129]: I would tend to say Reject or Decline
```suggestion
            {t('Reject')}
```

File: app/pages/terms/terms.js [hunk 3]
```diff
@@ -13,281 +13,173 @@
...
+            onChange={() => this.setState({ agreed: !this.state.agreed })} />
+          &nbsp;
+          <span>
+            {t('I am 18 or older and I accept the terms of the')}
+            &nbsp;
+            <a href={urlTermsOfUse} target='_blank' rel='noreferrer'>{textTermsOfUse}</a>
+            &nbsp;
+            {t('and the')}
+            &nbsp;
+            <a href={urlPrivacyPolicy} target='_blank' rel='noreferrer'>{textPrivacyPolicy}</a>
```
Comment (franck-fourel) [inline app/pages/terms/terms.js:102]: Not sure that kind of patterns can be translated correctly in any language if we don't use the Trans component.

File: locales/en/translation.json [hunk 4]
```diff
@@ -477,6 +473,7 @@
   "Ready!": "Ready!",
   "Recommended": "Recommended",
   "Refresh": "Refresh",
+  "Refuse": "Refuse",
```
Comment (franck-fourel) [inline locales/en/translation.json:476]: I would not use Refuse. Reject or Decline are probably more appropriate.

```suggestion
  "Reject": "Reject",
```

File: app/core/personutils.js [hunk 5]
```diff
@@ -24,7 +24,9 @@ const t = i18next.t.bind(i18next);
 
 import { MGDL_UNITS, MMOLL_UNITS } from './constants';
 
-let personUtils = {};
+let personUtils = {
+  haveClinicProfile: (user) => _.get(user, 'profile.clinic', false) !== false
```
[the remark this answers is not in the document]
Comment (yvon-dblg) [reply app/core/personutils.js:28]: I tried to have auto completion on vscode with this file... it failed

File: app/pages/terms/terms.js [hunk 6]
```diff
@@ -13,281 +13,173 @@
...
-    if (!this.props.termsAccepted) {
-      return null;
+  renderTermsTitle() {
+    const { termsWasAccepted } = this.props;
+    /** @type {string} */
+    const brand = _.get(CONFIG, `${config.BRANDING}.name`, 'Tidepool');
+
+    let termsTitle = null;
+    if (termsWasAccepted) {
+      this.log('Previous terms was accepted.');
```
[the remark this answers is not in the document]
Comment (yvon-dblg) [reply app/pages/terms/terms.js:50]: Not sure

**Pull request merged**